client/src/main/java/io/pravega/client/stream/EventWriterConfig.java

```diff
@@ -54,6 +55,7 @@
     private final boolean automaticallyNoteTime;
 
     public static final class EventWriterConfigBuilder {
+        private static final long MIN_TRANSACTION_TIMEOUT_TIME = 10000;
```

Comment:
nit: Could we rename the variable to indicate the timeout here is in seconds `MIN_TRANSACTION_TIMEOUT_TIME_SECONDS`?

Reply:
I want it to have the same units as the timeout below.

Reply:
OK, then can we give it a suffix of `MILLIS` so that we know what unit it is in?

Reply:
Done